Source/Core/Base.h

```diff
@@ -96,6 +97,8 @@ inline QString Helper::ToString<Core::AirPods::Model>(const Core::AirPods::Model
         return "AirPods Pro";
     case Core::AirPods::Model::AirPods_Pro_2:
         return "AirPods Pro 2";
+    case Core::AirPods::Model::AirPods_Pro_2_USB_C:
+        return "AirPods Pro 2(USB-C)";
```

Comment:
```suggestion
        return "AirPods Pro 2 (USB-C)";
```

Just nitpick, inserting a space would be better.